dockerfiles/gomoderator/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM golang:alpine as builder
+
+RUN apk --update --no-cache add make git g++ linux-headers python3
+
+ARG VDB_VERSION=staging
+ENV GO111MODULE on
+
+WORKDIR /go/src/github.com/makerdao
+RUN git clone https://github.com/makerdao/vulcanizedb.git
+RUN git clone https://github.com/makerdao/vdb-mcd-transformers.git
+WORKDIR /go/src/github.com/makerdao/vulcanizedb
+RUN git checkout $VDB_VERSION
+RUN go build
+WORKDIR /go/src/github.com/makerdao/vdb-mcd-transformers
+RUN git checkout $VDB_VERSION
```

Comment:
Does this seem right or would I want to run `vulcanizedb[staging] `vs. the `mcd` branch currently being built?

Reply:
I think you probably want to stick with the mcd branch that's currently being built so that the status indicates whether a given feature branch/PR introduces new dep conflicts